.github/workflows/ci_tests.yaml

```diff
@@ -0,0 +1,99 @@
+# This workflow will install Python dependencies, run tests and lint with a single version of Python
+# For more information see: https://help.github.com/actions/language-and-framework-guides/using-python-with-github-actions
+
+name: Tests
+
+on:
+  push:
+    branches: [ master ]
+  pull_request:
+    branches: [ master ]
```

Comment:
Does this mean only PRs for the master branch are tested? What will happen if I have a PR for another PR?

Reply:
Then the CI won't run. I suppose we could change it so that it runs for any Pull Request.